engines/director/lingo/lingo-code.cpp

```diff
@@ -1651,12 +1653,12 @@ void LC::call(const Symbol &funcSym, int nargs, bool allowRetVal) {
 		if (funcSym.u.bltin != LB::b_return && funcSym.u.bltin != LB::b_value) {
 			if (stackSize == stackSizeBefore + 1) {
 				if (!allowRetVal) {
-					warning("dropping return value");
+					warning("builtin %s dropping return value", funcSym.name->c_str());
```

Comment:
I would also print here the dropped value. Also please put `builtin '%s' dropping` for better readability.

Reply:
Fixed.